tools/transifex/.tx/config

```diff
@@ -89,3 +89,41 @@ trans.pt_BR = work-dir/messages_pt_BR.properties
 trans.ru_RU = work-dir/messages_ru_RU.properties
 trans.zh_CN = work-dir/messages_zh_CN.properties
 
+[CloudStack_UI.48xmessagesproperties]
```

Comment:
If it will be added in #1375, I think we should be removing it from this PR?  Once we get #1375 in, you will have to rebase and fix conflicts.  When the changes are added to 4.8, they will then be forward merged into master (4.9), so we will want to get #1375 in first and then rebase this PR against master after that PR has been merged into master.  Thanks...